x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -57,9 +60,13 @@ export const SessionView = ({ sessionEntityId, height, jumpToEvent }: SessionVie
   const hasData = data && data.pages.length > 0 && data.pages[0].events.length > 0;
   const renderIsLoading = isFetching && !data;
   const renderDetails = isDetailOpen && selectedProcess;
+
   const toggleDetailPanel = () => {
     setIsDetailOpen(!isDetailOpen);
   };
+  const handleOptionChange = (value) => {
```

Comment:
the `value` isn't descriptive enough, it's probably also expecting a type definition